src/components/atoms/button/button.scss

```
@@ -0,0 +1,37 @@
+@use "../../variables" as * ;
+
+.btn-b1{
+    align-items : center;
+    color : #ffffff;
+    background : #000000;
```

色の類は基本的にvariables.scssで定義されている変数を使うようにしてください。
```suggestion
    color : $white;
    background : $black;
```